libraries/chain_kv/include/b1/session/undo_stack.hpp

```diff
@@ -2,11 +2,21 @@
 
 #include <queue>
 
+#include <fc/filesystem.hpp>
+#include <fc/io/fstream.hpp>
+#include <fc/io/datastream.hpp>
+#include <fc/io/raw.hpp>
+#include <fc/reflect/typename.hpp>
+#include <fstream>
 #include <b1/session/session.hpp>
 #include <b1/session/session_variant.hpp>
 #include <eosio/chain/exceptions.hpp>
 
 namespace eosio::session {
+   const uint32_t undo_stack_magic_number = 0x30510ABC;
```

Comment:
Can we make all of these either `const` or `constexpr`?

Reply:
Changed to use constexpr.